redaxo/src/addons/metainfo/lib/handler/handler.php

```diff
@@ -293,8 +302,7 @@ public function renderMetaFields(rex_sql $sqlFields, array $epParams)
                     $inputValue['hour'] = date('H', $dbvalues[0]);
                     $inputValue['minute'] = date('i', $dbvalues[0]);
 
-                    /** @var rex_input_date|rex_input_datetime $rexInput */
-                    $rexInput = rex_input::factory($typeLabel);
+                    /* @var rex_input_date|rex_input_datetime $rexInput */
```

Comment:
Hier hat der cs-fixer einen normalen Kommentar draus gemacht, da hier die Variable nicht zugewiesen wird in der Zeile drunter.
Wird die Zeile noch benötigt?

Reply:
Hab die phpdocs entfernt

Reply:
Für mich ist die Zeile noch zu sehen im Diff.